dali/kernels/imgproc/flip_cpu.h

```diff
@@ -36,7 +39,7 @@ inline int GetOcvType(size_t channels) {
 
 template <typename Type>
 void OcvFlip(Type *output, const Type *input,
-                    size_t layers, size_t height, size_t width, size_t channels,
+                    size_t depth, size_t height, size_t width, size_t channels,
```

Comment:
Why? And what about frames?

Reply:
Why? 
`depth` seems a better name along with height and width.
What about frames?
Do we want flip the order of frames?